src/hotspot/share/nmt/mallocHeader.inline.hpp

```diff
@@ -1,6 +1,6 @@
 /*
  * Copyright (c) 2014, 2023, Oracle and/or its affiliates. All rights reserved.
- * Copyright (c) 2021, 2022 SAP SE. All rights reserved.
+ * Copyright (c) 2021, 2023 SAP SE. All rights reserved.
```

Comment:
The copyright change in this file is obsolete now.

Reply:
done, thanks